src/components/enterprise-user-subsidy/enterprise-offers/styles/index.scss

```diff
@@ -0,0 +1,5 @@
+.low-offers-balance-alert-with-cta {
+    .alert-message-content {
+        @extend .col-lg-9;
```

Comment:
Shame we have to override a class name internal to the Paragon component... but after toying around, doesn't seem like there's currently a better alternative for the desired outcome.